content/docs/command-reference/add.md

```diff
@@ -59,11 +59,9 @@ Summarizing, the result is that the target data is replaced by small
 [`.dvc` files](/doc/user-guide/dvc-files-and-directories#dvc-files) that can be
 easily tracked with Git.
 
-> Note that `.dvc` files can be considered _orphan stages_, because they have no
-> <abbr>dependencies</abbr>, only outputs. These are treated as _always changed_
-> by `dvc status` and `dvc repro`, which always executes them. See
-> [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) to learn
-> more about stages.
+> Note that `.dvc` files don't have <abbr>dependencies</abbr>, only outputs.
+> These are treated as _always changed_ by `dvc status` and `dvc repro`, which
+> always executes them.
```

Comment:
Hm, might be missing something, but what does `dvc add` have to do with `always changed` stages? `dvc status` on `dvc add`ed files won't ever show `always changed`.

Reply:
yeah, the whole note looks weird. Also, how is it relevant to this PR?

Reply:
This note was already there, it wasn't introduced in this PR. I rewrote a few words in it for some reason, probably I thought it was a very small update while re-reading this ref. in the context of the actual scope. I have rolled it back to the even wronger version now, but guys, I think it should be fine to make some tiny unrelated changes in PRs here and there. Is it really better to extract to a tiny PR that may later cause conflicts? Idk, please lmk.

Reply:
let's remove it complete since we touched it. That's the problem of doing small changes - they distract and slow everything down. In this example - we are upgrading some wrong text to another wrong text - and how should we review it then? Now since we started - we spent some time already on communication, on reading this, on trying to understand, etc.

It is really better to do granular PRs. Merge conflicts is not a problem at all - that's what git is made for - solving conflicts among other things. Also, we should be doing these small updates after the main issue is done. Main issue in this PR is very simple and we should be doing updates like this in hours, not days or months.

Reply:
Removed. I just want to state that this is new to me: we had previously agreed tiny (seemingly unrelated) updates can be included in PRs. I think it boils down to the fact that editing human language cannot be the same as with code. But OK, I will adhere to this going forward.

Reply:
they def can be included, but in this specific case it feels that >50% are not directly related.

I would prefer the workflow where we fix the issue as quick as possible first - like do 80% of the job in a few hours, merge it and then improve other things on top.